test/sql/wrapped_filesystems_info.test

```diff
@@ -0,0 +1,13 @@
+# name: test/sql/wrapped_filesystems_info.test
+# description: test wrapped filesystems info
+# group: [cache_httpfs]
+
+require cache_httpfs
+
+query I
+SELECT * FROM cache_httpfs_get_cache_filesystems();
+----
+cache_httpfs with HTTPFileSystem
```

Comment:
I think you need to sort cached filesystem names, otherwise it's a flaky test.
Also document the function return names ordering.